smart-contracts/test/helpers/multisig.js

```diff
@@ -3,7 +3,7 @@ const { impersonate } = require('./fork')
 
 const multisigABI = require('./ABIs/multisig.json')
 const UNLOCK_MULTISIG_ADDRESS = '0xa39b44c4AFfbb56b76a1BF1d19Eb93a5DfC2EBA9'
-const MULTISIG_ADDRESS_OWNER = '0xF5C28ce24Acf47849988f147d5C75787c0103534'
```

Comment:
Hum, Expected?

Reply:
Yes, that address has been removed from signers on mainnet